libcontainer/process_linux.go

```diff
@@ -513,6 +519,15 @@ func (p *initProcess) start() (retErr error) {
 			sentRun = true
 
 		case rootfsReady:
+			// Setup cgroup v2 child cgroup
```

Comment:
There's probably a good reason for this, but I'm curious, why can't this logic by applied above where you handle the cgroup-v1 cases?

Reply:
There is an important reason, but unfortunately I did not capture it in the comments and now I don't remember. Initially the child cgroup for both cgroups v1 and v2 was created at the same place in the code, but I had to move the cgroups v2 to a later point as otherwise things were not working properly.